src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncSpillTests.cs

```diff
@@ -3274,7 +3274,85 @@ public static async Task Boom()
 }
 ";
 
-            var v = CompileAndVerify(source, "System.Int32");
+            var verifier = CompileAndVerify(source, expectedOutput: "System.Int32",
+                verify: Verification.FailsILVerify with { ILVerifyMessage = "[MoveNext]: Unrecognized arguments for delegate .ctor. { Offset = 0x6d }" });
```

Comment:
Consider tagging this case with a ref to the runtime issue so readers have context. 

https://github.com/dotnet/runtime/issues/96695